dashbuilder/dashbuilder-shared/dashbuilder-displayer-api/src/main/java/org/dashbuilder/displayer/DisplayerSettings.java

```diff
@@ -44,6 +44,12 @@ public DisplayerSettings(DisplayerType displayerType) {
         this();
         setType(displayerType);
     }
+    
+    public DisplayerSettings(DisplayerType displayerType, DisplayerSubType subType) {
```

Comment:
Yes, that is possible with some displayers that have no subtype. I added this new constructor because I noticed that users could forgot to add a subtype and it would result in NPE!